spec/deploy_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
```

Comment:
Awesome Pants! Yay testing! Just fyi, I have found the http gem a lot easier to use than net/http, but it looks like you got it working!

If there's funkiness with connecting to the live server in CI, we may want to look at the Webmock gem

Reply:
While I was building out the other tests in this suite I refactored to use the http gem, you're right about it being easier.  

I will think about CI at another time.  :)